docs/_data/navs.yml

```diff
@@ -17,6 +17,8 @@ v1.0.x:
   - label: Pipeline hooks
   - label: The deployment context
   - label: Creating a plugin pack
+  - label: Creating an in-repo-addon plugin
```

Comment:
`Creating in-repo plugins`? Alternatively we could look at how to separate `label` and `link`